th2_data_services/data_source.py

```diff
@@ -8,17 +8,38 @@
 from datetime import datetime, timezone
 from csv import DictReader
 from typing import Generator, Iterable, List, Union, Optional
-from urllib.parse import urlencode
 from sseclient import SSEClient
-
+from th2_grpc_data_provider.data_provider_pb2 import Filter as grpc_Filter, FilterName as grpc_FilterName
 from th2_data_services.adapter import change_pipeline_message
 from th2_data_services.data import Data
-
+import google.protobuf.wrappers_pb2
 import logging
 
 logger = logging.getLogger('th2_data_services')
 logger.setLevel(logging.DEBUG)
 
+
+class Filter:
```

Comment:
1. Move the class to a separate module.
2. Filter should be available as "from th2_data_services import Filter"